tests/test.cpp

```diff
@@ -151,8 +151,37 @@ flatbuffers::unique_ptr_t CreateFlatBufferTest(std::string &buffer) {
 void AccessFlatBufferTest(const uint8_t *flatbuf, size_t length) {
 
   // First, verify the buffers integrity (optional)
-  flatbuffers::Verifier verifier(flatbuf, length);
```

Comment:
Keep this original test. Then put your test beneath it and make clear that this specifically tests the ability to compute the size from the data algorithmically.